test/core/channel/channel_stack_test.cc

```diff
@@ -32,10 +33,14 @@
 
 static grpc_error_handle channel_init_func(grpc_channel_element* elem,
                                            grpc_channel_element_args* args) {
-  EXPECT_EQ(args->channel_args->num_args, 1);
-  EXPECT_EQ(args->channel_args->args[0].type, GRPC_ARG_INTEGER);
-  EXPECT_STREQ(args->channel_args->args[0].key, "test_key");
-  EXPECT_EQ(args->channel_args->args[0].value.integer, 42);
+  int test_value = grpc_channel_args_find_integer(args->channel_args,
+                                                  "test_key", {-1, 0, INT_MAX});
+  EXPECT_EQ(test_value, 42);
+  // TODO(hork): replace the string with a macro. See TODO in channel_args.h
```

Comment:
If we need to use this in tests, I'd be fine with adding such a macro, as long as it's in a file in src instead of in include.

Reply:
Added in channel_args.h as `GRPC_INTERNAL_ARG_EVENT_ENGINE`